From 94810a91c48ac62daed55b4aec70d04a44c10795 Mon Sep 17 00:00:00 2001
From: Waldemar Brodkorb <wbx@openadk.org>
Date: Thu, 16 Feb 2017 20:22:21 +0100
Subject: [PATCH] arm: fix static linking issues

As reported by Buildroot developers these files causing static
linking issues. The original contribution with the ARM unwind-resume
rework and GNU libc sync was made before the combined libc change.
But the patch was applied later, after the libc change and
it seems the test coverage for static linking didn't catch it in
the regression testing. Remove the files.

Signed-off-by: Waldemar Brodkorb <wbx@openadk.org>
---
 libpthread/nptl/sysdeps/arm/pt-arm-unwind-resume.c | 2 --
 libpthread/nptl/sysdeps/arm/rt-arm-unwind-resume.c | 1 -
 2 files changed, 3 deletions(-)
 delete mode 100644 libpthread/nptl/sysdeps/arm/pt-arm-unwind-resume.c
 delete mode 100644 libpthread/nptl/sysdeps/arm/rt-arm-unwind-resume.c

diff --git a/libpthread/nptl/sysdeps/arm/pt-arm-unwind-resume.c b/libpthread/nptl/sysdeps/arm/pt-arm-unwind-resume.c
deleted file mode 100644
index fd0cec4..0000000
--- a/libpthread/nptl/sysdeps/arm/pt-arm-unwind-resume.c
+++ /dev/null
@@ -1,2 +0,0 @@
-__asm__ (".set __libgcc_s_init, pthread_cancel_init");
-#include <arm-unwind-resume.c>
diff --git a/libpthread/nptl/sysdeps/arm/rt-arm-unwind-resume.c b/libpthread/nptl/sysdeps/arm/rt-arm-unwind-resume.c
deleted file mode 100644
index 2641dc5..0000000
--- a/libpthread/nptl/sysdeps/arm/rt-arm-unwind-resume.c
+++ /dev/null
@@ -1 +0,0 @@
-#include <arm-unwind-resume.c>
-- 
2.1.4

